components/locale/Switcher.vue

```diff
@@ -0,0 +1,41 @@
+<script setup>
```

Comment:
Add som empty lines to make the script more readable.

Reply:
<img width="584" alt="Screenshot 2023-09-17 at 19 22 54" src="https://github.com/et-dagen/et-dagen/assets/113290010/4a8c03f9-77b3-418e-ba1b-992bc3d2ea53">
tried to change lang, but ended up with lots of errors?